dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java

```diff
@@ -1600,6 +1600,21 @@ public static ProgramRuleVariable createProgramRuleVariable( char uniqueCharacte
         return programRuleVariable;
     }
 
+    public static ProgramRuleVariable createProgramRuleVariableWithSourceType( char uniqueCharacter,
```

Comment:
Do you foresee someone else needing this? I just wonder what the rule of thumb is with what to put in the DhisConventienceTest which already contains a lot of helper methods. It's also hard to accommodate what "every" test setup needs. Just writing a few tests I already felt like the helpers were not fulfilling what I specifically wanted.

Reply:
yes i am forecasting some new features that might need utility methods like this.

Reply:
you could add them then when the new features are added 😋